ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -386,6 +420,12 @@ pub trait AuthenticationService: Send + Sync {
     async fn reset_user_auth_token(&self, email: &str) -> Result<()>;
     async fn password_reset(&self, code: &str, password: &str) -> Result<()>;
     async fn request_password_reset_email(&self, email: String) -> Result<Option<JoinHandle<()>>>;
+    async fn update_user_password(
+        &self,
+        email: &str,
```

Comment:
this probabaly can be changed to ID, but i can take the follow up PR after the id one got merged.

Reply:
Will do in follow-up, since ID will be included in the JWT then